workstation-backend/component/models.py

```diff
@@ -0,0 +1,36 @@
+from django.db import models
+from workspace import models as workspace_models
+
+# Create your models here.
+class BaseComponent(models.Model):
+    workspace_id = models.ForeignKey(workspace_models.Workspace, on_delete=models.CASCADE)
+    x = models.DecimalField(max_digits=100, decimal_places=2)
+    y = models.DecimalField(max_digits=100, decimal_places=2)
+    width = models.DecimalField(max_digits=100, decimal_places=2)
+    height = models.DecimalField(max_digits=100, decimal_places=2)
+
+    class Meta:
+        abstract = True
+
+
+# Checklist component
+class Checklist(BaseComponent):
+    title = models.CharField(max_length=200)
+    collapsed = models.BooleanField()
+    color = models.CharField(max_length=10, default="#baaf13")
+
+
+class ChecklistItem(models.Model):
+    checklist_id = models.ForeignKey(Checklist, on_delete=models.CASCADE)
+    content = models.CharField(max_length=200)
+    is_checked = models.BooleanField()
+
+
+class PostIt(BaseComponent):
+    content = models.CharField(max_length=200)
```

Comment:
I think we can make this longer since in the example UI, we can post a paragraph